jvm/src/main/scala/org/apache/spark/util/memory/TaskMemoryResources.scala

```diff
@@ -0,0 +1,151 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.util.memory
+
+import io.glutenproject.memory.TaskMemoryMetrics
+
+import org.apache.spark.{SparkEnv, TaskContext}
+import org.apache.spark.memory.TaskMemoryManager
+import org.apache.spark.sql.internal.SQLConf
+import org.apache.spark.util.TaskCompletionListener
+import org.apache.spark.util.memory.TaskMemoryResources._
+import scala.collection.JavaConverters._
+import java.util.concurrent.atomic.AtomicLong
+import java.util.UUID
+
+import org.apache.spark.internal.config.MEMORY_OFFHEAP_SIZE
+
+class TaskMemoryResources {
```

Comment:
remove this class, we only need the object TaskMemoryResources.

Reply:
removed.